packages/storybook/stories/components/toggle/documentation.mdx

```diff
@@ -0,0 +1,36 @@
+import { Canvas, Meta, Markdown } from '@storybook/blocks';
+import SpecificationsToggle from '@ovhcloud/ods-components/src/components/toggle/documentation/spec.md?raw';
+import { Banner } from '../../banner';
+import { DocNavigator } from '../../doc-navigator';
+import * as ToggleStories from './toggle.stories';
+
+<Meta of={ ToggleStories } name='Documentation' />
+
+<Banner of={ ToggleStories } />
+
+# Overview
+
+Toggle component is used to allow the user to switch between two states, on and off, in a visual representation:
+
+<Canvas of={ ToggleStories.Default } sourceState='none' />
+
+<DocNavigator of={ ToggleStories } />
+
+<Markdown>{ SpecificationsToggle }</Markdown>
+
+# Examples
+
+## Usage
```

Comment:
I think the template needs to be updated, some components have this `Usage` part, some don't
Can you remove it (and change children from `###` to `##`) from the template and update all files?